{
  callback.onError(Promise.getError("Connection closed"));
}
